tests/pages/analytics/index.py

```diff
@@ -0,0 +1,53 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+
+from selenium.webdriver.common.by import By
+
+from pages.base import BasePage
+
+
+class AnalyticsTestPage(BasePage):
+
+    _URL_TEMPLATE = "/{locale}/analytics/"
```

Comment:
```suggestion
    _URL_TEMPLATE = "/{locale}/analytics-tests/"
```

to match the change suggested above